nilearn/datasets/utils.py

```diff
@@ -62,6 +62,17 @@ def readlinkabs(link):
         return path
     return os.path.join(os.path.dirname(link), path)
 
+def pheno_decode(a):
+    """
+    Utility function for decoding byte strings in the data
+    """
+    a = np.array(a).tolist()
+    for i in xrange(len(a)):
+        for j in xrange(len(a[0])):
+            if isinstance(a[i][j], str):
+                    a[i] = list(a[i])
+                    a[i][j] = a[i][j].decode()
```

Comment:
Rather than a for loop, you should be using "np.char.decode" that you can apply on an array (eg a column of consistent type).